CSS/index.css

```diff
@@ -106,4 +111,103 @@ input#menuButton {
     transform: translateY(-305px);
     transition: transform 500ms;
   }
+
+  #bGImage {
+    margin-top: -213px;
+  }
+}
+/* Main Banner */
+#bGImage {
+  background-image: url("../assets/BGimage.png");
+  background-color: rgb(241, 241, 241);
+  height: 700px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  z-index: -1;
+
+  /* is this negative margin ok? */
+}
+
+.b_g_text {
+  text-align: center;
+  position: absolute;
+  font-size: 40px;
+  font-family: Arial, Helvetica, sans-serif;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  color: white;
+}
+
+h1 {
+  display: flex;
+  flex-direction: row;
+}
+
+h2 {
+  font-size: 80px;
+}
+.b_g_text_two {
+  left: 30%;
+  transform: translate(20%, -50%);
```

Comment:
I try to reserve transform for animations, id rather you use bottom: -50% in this case